.github/automation/.drone.yml

```diff
@@ -25,7 +42,7 @@ steps:
 - name: gcc-test
   image: ubuntu:16.04
   commands:
-  - apt-get update && apt-get install -y git build-essential cmake
+  - apt-get update && apt-get install -y git build-essential cmake scons
```

Comment:
nit: here and in other non-ACL builds -- scons is not used. what is the reason to add it?

Reply:
No reason at all! Just a mistake, thanks for pointing it out - I'll fix it.